lib/purserl.ex

```diff
@@ -167,13 +171,59 @@ defmodule DevHelpers.Purserl do
     nil
   end
 
+  def print_pretty_status(state, module) do
+    {pos, step_in_brackets, s_version} = state.module_positions[module]
+    rows = :maps.size(state.module_positions)
+    {new_line, label} =
+      case state.erl_steps[module] do
+        nil ->
+          {true, Color.format([:yellow, "Purs"])}
```

Comment:
Does this work for people who don't want color in their terminal? E.g. vim buffers and processing the file?

Reply:
I believe so. It seems to be automatically handled, which is nice. You can look at the CI output to verify

Reply:
This might not have been 100% correct. It really depends on `IO.ANSI.enabled?`, which is enabled if both stdout and stderr are detected to be terminals. This means that starting the compiler from, for example, a CI job will disable the color. This option can be easily overridden though when using format, so it might be something we want to do in the future